src/background/services/simple-storage copy.ts

```diff
@@ -1,31 +1,32 @@
-import log from 'loglevel';
-import { browser } from 'webextension-polyfill-ts'
+import log from "loglevel";
```

Comment:
Please let's delete this script I was just for testing something before, but now not needed.

Reply:
I'll fix it